examples/sourceview/test2.ml

```diff
@@ -27,7 +36,7 @@ let print_lang_dirs (language_manager:GSourceView3.source_language_manager) =
     (fun dir -> incr i; prerr_endline (sprintf "%d: %s" !i dir))
     language_manager#search_path
 
-let win = GWindow.window ~title:"LablGtkSourceView 2 test" ()
+let win = GWindow.window ~title:"LablGtkSourceView 3 test" ()
 let vbox = GPack.vbox ~packing:win#add ()
 let hbox = GPack.hbox ~packing:(vbox#pack ~expand: false) ()
 let bracket_button = GButton.button ~label:"( ... )" ~packing:hbox#add ()
```

Comment:
IMVHO it would be better if these let's where not in the toplevel but under a `let main () = _`, but indeed I understand this style exporting the global symbols was used before to run things under OCaml.

Reply:
agreed, i almost made that change and then decided to keep the changes minimal :) i'll do it.

Reply:
done